Sources/StreamChat/Models/Channel.swift

```diff
@@ -71,9 +71,9 @@ public struct _ChatChannel<ExtraData: ExtraDataTypes> {
     public var lastActiveMembers: [_ChatChannelMember<ExtraData.User>] { _lastActiveMembers }
     @CoreDataLazy private var _lastActiveMembers: [_ChatChannelMember<ExtraData.User>]
     
-    /// A list of currently typing channel members.
-    public let currentlyTypingMembers: Set<_ChatChannelMember<ExtraData.User>>
-
+    /// A list of currently typing users.
+    public let currentlyTypingUsers: Set<_ChatUser<ExtraData.User>>
```

Comment:
would be nice to have backing `@CoreDataLazy` field for this

Reply:
done 👍🏻 , please check

Reply:
Can you please add the `currentlyTypingMembers` variable into `Deprecations.swift` (with empty implementation) and mark it as renamed? It will help customers with the transition because they will be able to apply fixits.